pandas/io/common.py

```diff
@@ -318,7 +362,7 @@ def _get_handle(path_or_buf, mode, encoding=None, compression=None,
     """
     try:
         from s3fs import S3File
-        need_text_wrapping = (BytesIO, S3File)
+        need_text_wrapping = (BytesIO, S3File)  # type: Tuple
```

Comment:
Was this throwing a Typing error? Think MyPy should be able to infer here

Reply:
@WillAyd Yeah, it was. Couldn’t tell why, but MyPy couldn’t infer when I added types to the function definition. It happened in a couple places.

Reply:
What error was it giving you?

Reply:
It was giving `pandas/io/common.py:367: error: Incompatible types in assignment (expression has type "Tuple[Type[BytesIO]]", variable has type "Tuple[Type[BytesIO], Any]")` only if any of the function's arguments have type annotations.